### PR TITLE
fix: Better element ID naming for disambiguation (fixes mdn/web-components-examples#74)

### LINKED ISSUE #74: Improve readability for 'simple-template' example
My issue relate to: https://github.com/mdn/web-components-examples/tree/712ce27cd5b3c894ebc9b9958ec0973e3e86157b/simple-template

My point is the custom-element name and the template id name is the same that confused for anyone start to read from `main.js` file. I would like to replace `<template id="my-paragraph">` to `<template id="template-for-my-paragraph">`.

If you guy think it is a right way, I would like to make my PR to this repo.

### PR DESCRIPTION
**Pull request by ngdangtu-vn**
### Description

This is the sibling PR with https://github.com/mdn/web-components-examples/pull/77

### Motivation

To match the web-components-examples example.

### Additional details

none

### Related issues and pull requests

- Fixes mdn/web-components-examples#74
- Relates to https://github.com/mdn/web-components-examples/pull/77
- **Depends on:** [#123](https://github.com/mdn/web-components-examples/pull/77)

### CODE REVIEW

**Comment by ngdangtu-vn** [comment]
Hi @bsmth , I've updated the MDN doc example. Please take a look.

**Comment by bsmth** [comment]
Thanks @ngdangtu-vn - sorry for the delay, I've been away and could not take a look. I've added myself as a reviewer now!

File: files/en-us/web/api/web_components/using_templates_and_slots/index.md
Comment (bsmth) [inline files/en-us/web/api/web_components/using_templates_and_slots/index.md:1]: There's one more on L65 to get:

```diff
-<template id="my-paragraph">
+<template id="custom-paragraph">
```

**Comment by bsmth** [comment]
Thanks a lot! There's one leftover line to fix, but then we're ready to merge and the accompanying PR, too.

**Review by bsmth — CHANGES_REQUESTED** [review]
See comment about L65 in `files/en-us/web/api/web_components/using_templates_and_slots/index.md`

**Comment by bsmth** [comment]
Hey @ngdangtu-vn - thanks for your suggestions! Quick reminder there's one last thing to get on this branch and we're ready to go!

**Comment by bsmth** [comment]
@ngdangtu-vn - is it okay with you if I make the changes on your branch and merge?

**Comment by ngdangtu-vn** [comment]
@bsmth Sorry for later response, I think the the notification got lost somewhere. Hopefull it didn't pulling your task down too much.

**Review by bsmth — APPROVED** [review]
Super, looking good!

**Comment by bsmth** [comment]
> @bsmth Sorry for later response, I think the the notification got lost somewhere. Hopefull it didn't pulling your task down too much.

Not a problem, well done on your contribution!

**Comment by ngdangtu-vn** [comment]
> @ngdangtu-vn - is it okay with you if I make the changes on your branch and merge?

I don't at all. If it's convenience for you, please go ahead.

**Pull request merged**